config/plugin/network_problem.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
 # This plugin checks for common network issues. Currently, it only checks
-# if the conntrack table is full. 
+# if the conntrack table is full.
 
 OK=0
 NONOK=1
 UNKNOWN=2
 
-[ -f /proc/sys/net/ipv4/netfilter/ip_conntrack_max ] || exit $UNKNOWN
-[ -f /proc/sys/net/ipv4/netfilter/ip_conntrack_count ] || exit $UNKNOWN
+# "nf_conntrack" replaces "ip_conntrack" - support both
+NF_CONNTRACK_COUNT='/proc/sys/net/netfilter/nf_conntrack_count'
+NF_CONNTRACK_MAX='/proc/sys/net/netfilter/nf_conntrack_max'
+IP_CONNTRACK_COUNT='/proc/sys/net/ipv4/netfilter/ip_conntrack_count'
+IP_CONNTRACK_MAX='/proc/sys/net/ipv4/netfilter/ip_conntrack_max'
 
-conntrack_max=$(cat /proc/sys/net/ipv4/netfilter/ip_conntrack_max)
-conntrack_count=$(cat /proc/sys/net/ipv4/netfilter/ip_conntrack_count)
+if [ -f $NF_CONNTRACK_COUNT ] && [ -f $NF_CONNTRACK_MAX ]; then
+    CONNTRACK_COUNT=$NF_CONNTRACK_COUNT
+    CONNTRACK_MAX=$NF_CONNTRACK_MAX
+elif [ -f $IP_CONNTRACK_COUNT ] && [ -f $IP_CONNTRACK_MAX ]; then
+    CONNTRACK_COUNT=$IP_CONNTRACK_COUNT
+    CONNTRACK_MAX=$IP_CONNTRACK_MAX
+else
+    exit $UNKNOWN
+fi
+
+conntrack_count=$(< $CONNTRACK_COUNT) || exit $UNKNOWN
```

Comment:
nit: We have `conntrack_count` and `CONNTRACK_COUNT`, one for the value and another one for the path. do you mind appending `_PATH` to the path constants? Thanks so much!

Reply:
Not a nit, you're right about the naming.
I added `PATH` and shortened `CONNTRACK` to `CT`

Reply:
Sounds good to me! Thanks! :)